Orange/widgets/data/owrank.py

```diff
@@ -691,7 +691,6 @@ def create_scores_table(self, labels):
 
     @classmethod
     def migrate_settings(cls, settings, version):
-        super().migrate_settings(settings, version)
```

Comment:
OWWidget.migrate_settings would work. But what use would it really be? It can't do anything with the information provided by the version number (it's not its version).